examples/some-issuer/src/bin/discord.rs

```diff
@@ -303,9 +320,12 @@ async fn main() -> anyhow::Result<()> {
 
     {
         use tracing_subscriber::prelude::*;
+        let log_filter = tracing_subscriber::filter::Targets::new()
+            .with_target(module_path!(), app.log_level)
+            .with_target("tower_http", app.log_level);
```

Comment:
I don't think hyper is the right thing though. The reason they are ignored is that they are in the library component, so their target is `some-issuer` rather than `discord` or `telegram`.

I've added 
```
            .with_target(env!("some_issuer"), app.log_level)
```